nilearn/version.py

```diff
@@ -1 +1,97 @@
+# *- encoding: utf-8 -*-
+"""
+nilearn version, required package versions, and utilities for checking
+"""
+# Author: Alexandre Abraham, Philippe Gervais, Ben Cipollini
+# License: simplified BSD
+
 __version__ = '0.1b1'
+
+# All metadata needed for required modules:
+#    * minver: minimum required version of the module
+#    * manual: whether the module must be installed manually
+# See more info in check_module_dependencies docstring.
+#
+# This is a list to preserve order, and to avoid the Python 2.7
+#   dependency of collections.OrderedDict
+_NILEARN_INSTALL_MSG = 'See %s for installation information.' % (
+    'http://nilearn.github.io/introduction.html#installation')
+REQUIRED_MODULE_METADATA = (
+    ('numpy', {
+        'minver': '1.6.0',
+        'manual_install': True,
+        'install_info': _NILEARN_INSTALL_MSG}),
+    ('scipy', {
+        'minver': '0.9.0',
+        'manual_install': True,
+        'install_info': _NILEARN_INSTALL_MSG}),
+    ('sklearn', {
+        'minver': '0.10',
+        'manual_install': True,
+        'install_info': _NILEARN_INSTALL_MSG}),
+    ('nibabel', {
+        'minver': '1.1.0',
+        'manual_install': False}),
+    ('gzip', {
+        'minver': '0.0.0',
+        'manual_install': True,
+        'install_info': 'Use a python version compiled with gzip.'}))
```

Comment:
Just as a reminder, not having gzip support will result in an exception not a warning. You won't be able to fetch any datasets because you won't be able to import nilearn. Feel free to suggest a better error message!